### PR TITLE
Portfolio page

### PR DESCRIPTION
**Pull request by ellischung**
Finished implementing the portfolio page along with its components.

### CODE REVIEW

File: app/components/Portfolio/Asset.tsx [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+            )}
+            <span
+              className={
+                asset.profit_percentage > 0
+                  ? "text-[#00b1a6]"
+                  : "text-[#fe2264]"
+              }
+            >
+              {asset.profit_percentage}%
+            </span>
```
Comment (yoloOnTheBattlefield) [inline app/components/Portfolio/Asset.tsx]: you should make this a component

File: app/components/Portfolio/Asset.tsx [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+          <div className="flex items-center">
+            <span
+              className={`text-xl mr-6 ${
+                asset.price_change_percentage_24h_in_currency > 0
+                  ? "text-[#00b1a6]"
+                  : "text-[#fe2264]"
+              }`}
+            >
+              {asset.market_vs_volume}%
+            </span>
```
Comment (yoloOnTheBattlefield) [inline app/components/Portfolio/Asset.tsx]: cause you repeat it here

File: app/components/Portfolio/Modal.tsx [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
+          })
+        ).unwrap();
+
+        // Finalized asset to be created or edited
+        const newAsset: AssetData = {
+          id: assetToEdit ? assetToEdit.id : uuidv4(), // edited asset's id or generate new one
+          symbol: selectedCoin.symbol,
+          name: selectedCoin.name,
+          image: selectedCoin.image,
+          total_value: purchasedAmount,
```
Comment (yoloOnTheBattlefield) [inline app/components/Portfolio/Modal.tsx]: you could also get rid og the snake case if you are creating a new object

File: app/pages/portfolio/page.tsx [hunk 4]
```diff
@@ -1,3 +1,88 @@
...
+
+  return (
+    <div className="bg-[#f3f5f9] dark:bg-[#13121a] w-[90%]">
+      <div className="flex justify-between p-4 mt-4">
+        <div className="flex items-center text-[#3c3c7e] dark:text-white text-2xl">
+          Portfolio
+        </div>
+        <button
+          className="flex items-center bg-[#aaabe8] dark:bg-[#3c3c7e] text-[#3c3c7e] dark:text-white border-[#6161cb] shadow-whiteShadow rounded-md p-3 px-20"
+          onClick={() => openModal()} // call without params for 'ADD' modal
```
Comment (yoloOnTheBattlefield) [inline app/pages/portfolio/page.tsx]: onClick={openModal} // call without params for 'ADD' modal

**Review by yoloOnTheBattlefield — CHANGES_REQUESTED** [review]

File: app/components/Portfolio/Asset.tsx (hunk 1 above)
Comment (ellischung) [reply app/components/Portfolio/Asset.tsx]: Ok, I've made a reusable component for it.

File: app/components/Portfolio/Modal.tsx (hunk 3 above)
Comment (ellischung) [reply app/components/Portfolio/Modal.tsx]: Converted all to camelCase.

File: app/pages/portfolio/page.tsx (hunk 4 above)
Comment (ellischung) [reply app/pages/portfolio/page.tsx]: Made the requested change. Directly calls openModal now.

**Comment by ellischung** [comment]
I have implemented the requested changes and pushed the updates. Please review them when you can!

**Pull request merged**